### PR TITLE
examples: make output-power-management oneshot by default

### PR DESCRIPTION
**Pull request by ammen99**
As discussed in IRC, with oneshot mode, we can use this example client as a keybinding or anything like that. Otherwise, the user will have to manually kill the client each time.

### CODE REVIEW

File: examples/output-power-management.c [hunk 1]
```diff
@@ -72,22 +73,27 @@ static const struct wl_registry_listener registry_listener = {
 static const char usage[] = "usage: output-power-management [options...]\n"
 	"  -h: show this help message\n"
 	"  -e: turn outputs on\n"
-	"  -d: turn outputs off\n";
+	"  -d: turn outputs off\n"
+	"  -m: turn monitor mode on\n";
```
Comment (emersion) [inline examples/output-power-management.c:77]: "monitor mode" is a little bit confusing for a tool that configures outputs. Maybe we can change the description to "don't exit and monitor changes"?

Comment (ammen99) [reply examples/output-power-management.c:77]: what about "don't exit and continuously print changes" ?

Comment (emersion) [reply examples/output-power-management.c:77]: Sure, looks good!

Comment (agx) [reply examples/output-power-management.c:77]: or '-w': continousely watch for power mgmt mode chnges. Apart from that looks good to me.

Comment (ammen99) [reply examples/output-power-management.c]: I made it as @agx suggested (with -`w`)

**Comment by ammen99** [comment]
> LGTM, can you squash these changes?

Yes, done :)

**Comment by emersion** [comment]
Thanks!

**Pull request merged**